src/db/sysdb_ops.c

```diff
@@ -84,9 +86,49 @@ int sss_ldb_modify_permissive(struct ldb_context *ldb,
         return ret;
     }
 
+    ret = ldb_transaction_start(ldb);
+    if (ret != LDB_SUCCESS) {
+        sysdb_ret = sysdb_error_to_errno(ret);
```

Comment:
I think this is a copy-n-paste error from somewhere else, but this function should be returning a LDB error code, not errno or sssd-specific error code.

Reply:
(see the comment block near the function definition btw)

Reply:
I'm only using the sysdb_ret to properly log the error. I'm not returning it.
I'll change it, though, and re-submit the PR.